src/components/Button/Button.tsx

```diff
@@ -141,6 +141,70 @@ export const useStyles = makeStyles(
         opacity: 0.44,
       },
     },
+    containedNegative: {
+      backgroundColor: theme.palette.error.main,
+      color: theme.palette.common.white,
+      '&:hover': {
+        backgroundColor: theme.palette.error[900],
+      },
+      '&:disabled, &[disabled]': {
+        backgroundColor: theme.palette.error.main,
+        color: theme.palette.common.white,
+      },
+    },
+    containedPositive: {
+      backgroundColor: theme.palette.green.main,
+      color: theme.palette.common.white,
+      '&:hover': {
+        backgroundColor: theme.palette.green[900],
```

Comment:
Just a thought, but it might be worth introducing a `theme.palette.primary.positive` and `theme.primary.primary.negative` to make it easier to override these colors. Thinking about skillspring, where the primary color is green.

Reply:
✅ I took a stab at it. I might need some more direction if my latest changes don't provide what you're expecting.